auth/password-visibility.js

```diff
@@ -0,0 +1,15 @@
+const visible = document.getElementsByClassName('visible');
+
+function passwordVisibility (e) {
+    if (e.target.getAttribute("src") === "/img/visibility_off.png") {
+        e.target.setAttribute("src", "/img/visibility_on.png");
+        e.target.parentElement.previousElementSibling.setAttribute("type", "text");
+    } else {
+        e.target.setAttribute("src", "/img/visibility_off.png");
+        e.target.parentElement.previousElementSibling.setAttribute("type", "password");
+    }
+}
```

Comment:
💊 제안
반복사용되는 e.target을 변수에 저장해 아래처럼 가독성을 높일 수 있습니다.

```suggestion
function passwordVisibility (e) {
    const toggleButton = e.target;
    const input = e.target.parentElement.previousElementSibling;
    
    if (toggleButton.getAttribute("src") === "/img/visibility_off.png") {
        toggleButton.setAttribute("src", "/img/visibility_on.png");
        input.setAttribute("type", "text");
    } else {
        toggleButton.setAttribute("src", "/img/visibility_off.png");
        input.setAttribute("type", "password");
    }
}
```